MapboxDirections/MBDirections.swift

```diff
@@ -36,10 +37,7 @@ public class MBDirections: NSObject {
 
     private let request: MBDirectionsRequest
     private let configuration: MBDirectionsConfiguration
-    private var task: NSURLSessionDataTask?
-    public var calculating: Bool {
```

Comment:
We should keep this property for compatibility with MapKit. It's how client code knows that the object is already waiting for a request. Perhaps not the most elegant approach, but we should be deliberate about the places we diverge from MapKit.